src/test/java/utils/seleniumWrapper.java

```diff
@@ -0,0 +1,47 @@
+package utils;
+
+import org.junit.Assert;
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.edge.EdgeDriver;
+
+public class seleniumWrapper {
+
+    seleniumWrapper wrapper = new seleniumWrapper();
+
+public WebDriver driver = new EdgeDriver();
+
+       public boolean clickElement(By locator){
+        driver.findElement(locator).click();
+        return true;
+
+    }
+
+    //function 2 - isDisplayed
+    public boolean checkElementDisplay(By locator){
+       boolean flag=driver.findElement(locator).isDisplayed();
+        return flag;
+    }
```

Comment:
Could you please rename the function like this:
public booelan isDisplayed(By locator)

Similar for other functions